test/inspector_service_test.dart

```diff
@@ -0,0 +1,726 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@TestOn('vm')
+import 'dart:async';
+import 'dart:io';
+
+import 'package:devtools/inspector/diagnostics_node.dart';
+import 'package:devtools/inspector/flutter_widget.dart';
+import 'package:test/test.dart';
+
+import 'package:devtools/globals.dart';
```

Comment:
It looks like some portion of the devtools codebase will need to run in the VM (not an issue, just noting it). I expect that the tests on travis will keep us honest, if we start referencing dart:html code from portions of the app that need to be able to run on the cli.

Reply:
yes. these tests along with the one @kenzieschmoll already added keep us honest about not adding dart:html imports to our business logic code which is a great thing. When I'd first added the inspector code but let a few dart:html tests sneak in, @kenzieschmoll's tests started failing locally with complains about my deart:html deps.